l10n_es_facturae_face/tests/test_facturae_face.py

```diff
@@ -23,6 +25,7 @@ class EDIBackendTestCase(
 ):
     @classmethod
     def setUpClass(cls):
+        cls._super_send = requests.Session.send
```

Comment:
I think this is not necessary either

Reply:
It is necessary so external calls don't get blocked.